acme_diags/driver/utils/dataset.py

```diff
@@ -156,6 +156,20 @@ def get_climo_variable(self, var, season, extra_vars=[], *args, **kwargs):
         return variables[0] if len(variables) == 1 else variables
 
 
+    def get_static_variable(self, static_var, primary_var):
+        # TODO: is this function coded correctly? Need to do anything to deal with different resolutions/grids?
```

Comment:
Does the implementation for `get_static_variable` seem fine for this specific use case?

Reply:
The implementation looks good. Suggest to use `with` statement to open the file, so that the file handler doesn't need to be closed with a close statement.